.github/workflows/integration.yml

```diff
@@ -43,3 +43,14 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run W3C Trace Context docker-compose.integration
         run: docker-compose --file=test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/docker-compose.yml --file=build/docker-compose.${{ matrix.version }}.yml --project-directory=. up --exit-code-from=tests --build
+
+  otlp-exporter-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [netcoreapp2.1,netcoreapp3.1]
```

Comment:
@eddynaka We should probable get these up on net5.0 do you know if that support is there on actions yet?

Reply:
I can take this. I was just planning to work on that when updating an old PR earlier today. Created an issue for this: #1686.

Reply:
I can take this. I was just planning to work on that when updating an old PR earlier today. Created an issue for this: #1686.